tests/test_analog_signals.py

```diff
@@ -64,6 +83,11 @@ def test_acromag_readback(fake_acromag):
     assert fake_acromag.ai1_13.get() == 1.0
 
 
+def test_acromag_ch_readback(fake_acromag_in, fake_acromag_out):
+    assert fake_acromag_in.channel.get() == 0.23
```

Comment:
Do you need an is_close or smth to deal with the float equivalence? I'm not sure

Reply:
I am assuming I should not need an `is_close` or something of this sort, because i'm testing it against the exact value that I put it, but I could be wrong here

Reply:
Right, I guess there is no calculation going on, so you should me good 👍